tests/acceptance/pageObjects/FilesPageElement/fileActionsMenu.js

```diff
@@ -186,7 +186,7 @@ module.exports = {
       selector: '//button[@aria-label="Rename"]',
       locateStrategy: 'xpath'
     },
-    deleteImmediatelyButtonInFileRow: {
+    deleteImmediatelyButtonInAccordion: {
```

Comment:
This element is same as the `deleteButtonInAccordion` element in line number 165. May be we can think of a way about  using only one of them. For that the FileAction should be checked. @individual-it what do you think about this?

Reply:
we can also change
`this.performFileAction(this.FileAction.deleteImmediately)`
to 
`this.performFileAction(this.FileAction.delete)`

also clean the `deleteImmediately` occurances

Reply:
We can do that in a different PR. For now it is good to get the test scenario enabled again.